projects/packages/forms/src/contact-form/class-contact-form-plugin.php

```diff
@@ -493,6 +493,19 @@ public static function gutenblock_render_field_consent( $atts, $content ) {
 		return Contact_Form::parse_contact_field( $atts, $content );
 	}
 
+	/**
+	 * Render the file upload field.
+	 *
+	 * @param array  $atts - the block attributes.
+	 * @param string $content - html content.
+	 *
+	 * @return string HTML for the file upload field.
+	 */
+	public static function gutenblock_render_field_file( $atts, $content ) {
```

Comment:
beside changing this. We will also have to change the markup of the form html and add 
enctype="multipart/form-data"